src/maxminddb.c

```diff
@@ -281,6 +281,10 @@ LOCAL int map_file(MMDB_s *const mmdb)
         goto cleanup;
     }
     mmh = CreateFileMappingA(fd, NULL, PAGE_READONLY, 0, size, NULL);
+    if (mmh == NULL) {  /* Microsoft documentation for CreateFileMapping indicates this returns NULL not INVALID_HANDLE_VALUE on error */
```

Comment:
I saw that as well in other parts of the code.  However, I was trying to stay consistent with the "if" checks on the lines just before this code and just after this code, which puts the variable first, and the check value second.

However, I appreciate the logic behind putting the NULL first, and have updated and pushed that to my pull request.